### PR TITLE
add some sentry logging for activity around okta service calls

### PR DESCRIPTION
**Pull request by erluti**
Fully intend to roll this back, but adding some sentry-info logging in order to see if I can glean some more information about the reason connected apps aren't refreshing immediately.

### CODE REVIEW

File: app/models/okta_redis/grants.rb [hunk 1]
```diff
@@ -8,34 +8,51 @@ module OktaRedis
   class Grants < Model
     CLASS_NAME = 'GrantsService'
 
+    after_initialize do |grants|
+      grants.instance_variable_set(:@activity_log, [])
```
Comment (erluti) [inline app/models/okta_redis/grants.rb:12]: the `Model` class uses `Common::RedisStore` which defines the `after_initialize` hook.  This is mostly to tap into the "constructor" even though, which is in `RedisStore` which has a complex signature and I took this approach to be less invasive.

File: app/models/okta_redis/grants.rb [hunk 2]
```diff
@@ -8,34 +8,51 @@ module OktaRedis
...
+    end
+
     def all
-      okta_response.body
+      all_grants = okta_response.body
+      log_message_to_sentry('Getting all grants', :info,
```
Comment (erluti) [inline app/models/okta_redis/grants.rb:17]: `okta_response` will add some data too `@activity_log` and then we'll log all of that to sentry before returning the okta_response.body

File: app/models/okta_redis/grants.rb [hunk 3]
```diff
@@ -8,34 +8,51 @@ module OktaRedis
...
       end
     end
 
     private
 
     def okta_response
-      do_cached_with(key: cache_key) do
+      @activity_log << 'Get okta response including grants'
+      response = do_cached_with(key: cache_key) do
+        @activity_log << 'Fetch new response'
```
Comment (erluti) [inline app/models/okta_redis/grants.rb:50]: this is the issue we're ultimately trying to get insight on.  After deleting a grant, we should need to fetch a new response.  BUT if it's something with Okta's service, it wouldn't have changed that much

**Review by kfrz — APPROVED** [review]
This looks good to me!

**Pull request merged**